src/app/views/recommendations/_recommendation.html.erb

```diff
@@ -0,0 +1,6 @@
+<article class="panel has-background-white">
+  <p class="panel-heading">
+    <%= recommendation_title %>
+  </p>
+  <%= render functions %>
+</article>
```

Comment:
we need a new line here, configure your editor to make sure there is a new line at the end of the file